doc/installing/cluster.md

```diff
@@ -0,0 +1,83 @@
+# Running the OpenQuake Engine on multiple nodes (cluster configuration)
+
+## Overall architecture
+The nodes must all be able to communicate with the OpenQuake Engine *DbServer* and a *RabbitMQ* server.
+Both services run on a single "master" node. It is not necessary and not normally recommended to run *RabbitMQ* on a worker node and *Celery* on master node.
+
+## Initial install
+On all nodes, install the `python-oq-engine package` as described in OpenQuake Engine installation for [Ubuntu](ubuntu.md) or [RedHat](rhel.md).
+
+Note: you have to **restart every celery node** after a configuration change.
+
+## OpenQuake Engine 'master' node configuration File
+
+### Enable Celery
+
+In all the nodes, the following file should be modified to enable the *Celery* support:
+
+`/etc/openquake/openquake.cfg:`
+
+```
+[celery]
+# enable celery only if you have a cluster
+use_celery = true
+```
+
+## OpenQuake Engine 'worker' node configuration File
+On all worker nodes, the `/etc/openquake/openquake.cfg` file should be also modified to set the *DbServer* and *RabbitMQ* daemons IP address:
+
```

Comment:
Should we add something like `Please consider putting the configuration file on a shared file system so that there is only one of it` ?